packages/flame/test/game/transform2d_test.dart

```diff
@@ -7,6 +7,59 @@ import 'package:test/test.dart';
 import 'package:vector_math/vector_math.dart';
 
 void main() {
+  // Implementation of statistical error propagation based on
+  // Tellinghuisen, J. (2001). Statistical Error Propagation.
+  // https://pubs.acs.org/doi/10.1021/jp003484u
+  // @note, this function is only applicable to the globalToLocal 2D transform
```

Comment:
`@note` doesn't do anything in dartdocs afaik

```suggestion
  // Note: this function is only applicable to the globalToLocal 2D transform
```